firebase-document.html

```diff
@@ -86,7 +86,7 @@
           reject(new Error('No app configured!'));
         }
 
-        if (key) {
+        if (key !== undefined) {
```

Comment:
Please use `typeof key !== 'undefined'` instead.

Reply:
This has been done

Reply:
It’s locally defined as a function parameter so it’s perfectly fine to use direct comparison here.